readme.txt

```diff
@@ -2,10 +2,11 @@
 
 
 == Code Analysis==
-
-1. PHPStan is being used for running static code analysis. It's configuration file 'phpstan.neon' is available is this repository. The directories mentioned under scnDirectories option in phpstan.neon, are required for running the analysis. These directories belong to prestashop. If you don't have prestashop downloaded, you'll need to download it first and then make sure the paths are correctly reflected in phpstan.neon file. Once done, we can run the analysis: 
-  i.  First install composer packages using 'composer install'
-  ii. Then run '_PS_ROOT_DIR_=./ php vendor/phpstan/phpstan/phpstan.phar --configuration=phpstan.neon analyse ./' to run the analysis. It'll print out any errors detected by PHPStan.
-
-2. Php-CS-Fixer is being used for fixing php coding standard relared issues. After installing composer packge, simply run 'php vendor/bin/php-cs-fixer fix --dry-run --diff' and it'll print out any issues detected which you can automatically fix by running command 'php vendor/bin/php-cs-fixer fix' 
+PhpStan and Php-CS-Fixer tools are being used for static code analysyis and verifying php coding standards. Follow the below mentioned steps to run these tools
+1. Download PrestaShop source code from https://download.prestashop.com/download/releases/prestashop_1.7.7.0.zip and extract it.
```

Comment:
Where should I extract it to and do I need to adjust _PS_ROOT_DIR_ to match that somehow?

Reply:
extract it anywhere, and PS_ROOT_DIR is already being set in the command mentioned

Reply:
Ok, make a note that the person running the tool need to adjust the command so that PS_ROOT_DIR points to where it was extracted.